### PR TITLE
Update DND sensor icon based on state

### PR DESCRIPTION
**Pull request by mikosoft83**
## Summary
Adding icons based on DND sensor state. Makes for easier visual aid.

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

**Comment by jpelgrom** [comment]
Not sure I agree with this change as modern Android versions use DND as on/off (switching between priority_only and off). The icons you're using sort of resemble ringer mode.

**Comment by mikosoft83** [comment]
Would it make sense then to clump all DND modes together using single icon for "on" and another for "off"?

**Comment by jpelgrom** [comment]
If there must be an icon change yes, but as said I don't think the icon should be changed as the system doesn't change icons depending on on/off.

**Comment by mikosoft83** [comment]
At least in my version of android, no, the icon itself doesn't change but the background changes as a visual indication.
And in HA itself the binary sensors often have different icons for on and off state.

File: common/src/main/java/io/homeassistant/companion/android/common/sensors/DNDSensorManager.kt [hunk 1]
```diff
@@ -72,12 +72,16 @@ class DNDSensorManager : SensorManager {
...
+            "off" -> "mdi:minus-circle-off"
+            else -> "mdi:minus-circle"
+        }
 
         onSensorUpdated(
             context,
             dndSensor,
             state,
-            dndSensor.statelessIcon,
+            icon,
```
Comment (dshokouhi) [inline common/src/main/java/io/homeassistant/companion/android/common/sensors/DNDSensorManager.kt:84]: You can simplify this and remove the need for the `icon` variable defined above

```suggestion
            if (state == "off") dndSensor.statelessIcon else "mdi:minus-circle",
```

**Review by dshokouhi — APPROVED** [review]

**Pull request merged**